questions.md

```diff
@@ -0,0 +1,4 @@
+### Pull Requests
+
+* What's the difference between a "pull request" and a "branch"?
+* How do we handle conflict during a pull request?
```

Comment:
So glad you asked! Here is a tutorial on merge conflicts in the classroom: https://github.blog/2018-08-22-merge-conflicts-in-the-classroom/

Reply:
Thank you! 💯